locale/manageiq.pot

```diff
@@ -54291,6 +54388,19 @@ msgid_plural "Physical Server Profile (CiscoIntersight)"
 msgstr[0] ""
 msgstr[1] ""
 
+#: ../config/model_display_names.rb
+msgid "Physical Server Profile Template"
+msgid_plural "Physical Server Profile Templates"
+msgstr[0] ""
+msgstr[1] ""
+
+#:
+#: ../app/models/manageiq/providers/cisco_intersight/physical_infra_manager/physical_server_profile_template.rb
+msgid "Physical Server Profile Template (CiscoIntersight)"
+msgid_plural "Physical Server Profile Template (CiscoIntersight)"
```

Comment:
@agrare Should there be a space here?

```suggestion
msgid "Physical Server Profile Template (Cisco Intersight)"
msgid_plural "Physical Server Profile Template (Cisco Intersight)"
```

Reply:
:+1: yes that comes from `manageiq-providers-cisco_intersight/app/models/manageiq/providers/cisco_intersight/physical_infra_manager/physical_server_profile_template.rb` I'll open a PR to fix

Reply:
https://github.com/ManageIQ/manageiq-providers-cisco_intersight/pull/85

Reply:
Merged